services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -22,7 +23,11 @@ export const AppRoutes = () => {
         <Route path="/report/QM" element={<CreateReportOptions />} />
         <Route
           path="/report/:reportType/:state/:reportId"
-          element={<ReportPageWrapper></ReportPageWrapper>}
+          element={<ReportPageWrapper />}
```

Comment:
I believe `<Route> `can be self closing similar to how you did with `<ReportPageWrapper />` and how it's written in the lines above for other routes.

Reply:
good catch! fixed